### PR TITLE
refactor(explorer): transaction route now uses explored

### PR DESCRIPTION
**Pull request by telestrial**

### CODE REVIEW

**Comment by telestrial** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/SiaFoundation/web/740?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#740** <a href="https://app.graphite.dev/github/pr/SiaFoundation/web/740?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#739** <a href="https://app.graphite.dev/github/pr/SiaFoundation/web/739?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @telestrial and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/explorer/app/tx/[id]/page.tsx [hunk 1]
```diff
@@ -23,21 +23,40 @@ export const revalidate = 0
...
+    <Transaction
+      transaction={transaction}
+      transactionHeaderData={{
+        id: stripPrefix(transaction.id),
+        blockHeight: transactionChainIndices[0].height,
+        confirmations: currentTip.height - transactionChainIndices[0].height,
+        timestamp: parentBlock.timestamp,
+      }}
+    />
+  )
```
Comment (alexfreska) [inline apps/explorer/app/tx/[id]/page.tsx:61]: Nice with the parallelizing where possible and also processing and passing minimal required data from the server. 👍🏼

File: apps/explorer/components/Transaction/index.tsx [hunk 2]
```diff
@@ -87,33 +95,33 @@ export function Transaction({ title, transaction }: Props) {
...
     })
-    transaction.host_announcements?.forEach((host) => {
-      operations.push({
-        label: 'host announcement',
-        // type: 'host',
-        hash: host.net_address,
-      })
-    })
-    transaction.storage_proofs?.forEach((proof) => {
+    // transaction.host_announcements??.forEach((host) => {
```
Comment (alexfreska) [inline apps/explorer/components/Transaction/index.tsx:114]: Will explored not have these?

**Review by alexfreska — COMMENTED** [review]
Looks great!! lgtm, just have a type/build error to fix - you can recreate/check for these locally with `nx build explorer`.
Screenshot 2024-09-12 at 2 33 27 PM

**Comment by telestrial** [comment]
I elected to do some temporary additional logic on the contract route side to accommodate this phase out. I think even doing the transactions in that route through explored would have cascaded to a good deal of other changes that confound the transaction and contract routes a bit.

File: apps/explorer/app/contract/[id]/page.tsx [hunk 3]
```diff
@@ -76,13 +77,45 @@ export default async function Page({ params }) {
   const renewalTransaction = rt?.transaction
   const renewedTo = renewalTransaction?.storage_contracts?.[0]
 
+  // The following is a temporary addition to satisfy new Transaction
```
Comment (alexfreska) [inline apps/explorer/app/contract/[id]/page.tsx:80]: Whats the plan with this temporary addition, swap everything out in a follow up PR?

File: apps/explorer/components/Transaction/index.tsx (hunk 2 above)
Comment (alexfreska) [reply apps/explorer/components/Transaction/index.tsx:114]: We did not get an answer on this, all good for this PR, but maybe record a todo on this and we can follow up on the discussion thread.

File: apps/explorer/app/contract/[id]/page.tsx (hunk 3 above)
Comment (telestrial) [reply apps/explorer/app/contract/[id]/page.tsx:80]: Correct, yes. Like all the routes so far, there is a fair amount of reconciliation that needs to occur between what Explorer asks for from Sia Central for a contract and what it presently can receive from explored for a contract.

This much code satisfies the Transaction component within ContractView. Another option is to temporarily remove that Transaction component from ContractView,

**Review by alexfreska — APPROVED** [review]

**Comment by telestrial** [comment]
### Merge activity

* **Sep 16, 11:17 AM EDT**: @telestrial started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/SiaFoundation/web/740).

**Pull request merged**